tests/search_memory_index.cpp

```diff
@@ -295,24 +302,30 @@ int main(int argc, char** argv) {
     return -1;
   }
 
+  if (fail_if_recall_below < 0.0 || fail_if_recall_below >= 100.0) {
```

Comment:
Do you intend this to fail if fail_if_recall_below == 100.0 ?  The message indicates the value should be inclusive of 100%

Reply:
Yes. 100% recall is not something we should promise with this tool.